arc/usr/share/asteroid-launcher/watchfaces/Arc-black-pie-noseconds.qml

```diff
@@ -0,0 +1,11 @@
+Arc {
```

Comment:
I don't see where all those Arc-*.qml items are being used, could you clarify?

Reply:
Those are variants that just look a bit different – black vs. white; arc vs. pie chart; seconds visible vs. not visible. They are listed on their own in the watchfaces settings.

Reply:
Fair enough, I didn't understand directly. This can be merged as well